content/pt/video-courses/index.md

```diff
@@ -1,64 +1,64 @@
 ---
 template: blog
-title: 'Video Courses'
-description: 'Discover and learn more about the Nuxt Framework with free and premium video courses.'
+title: 'Cursos em Vídeo'
+description: 'Descubra e aprenda mais sobre o abstração Nuxt com cursos em vídeos gratuitos e prémios.'
 ---
 ::video-courses-list
 ---
 video-courses:
   one:
-    title: Learn Nuxt by Building a Real World App
-    description: Learn how to build robust, modern websites with Nuxt from scratch. Or improve your website performance, code quality, while making better use of the framework.
+    title: Aprenda Nuxt Construindo uma Aplicação do Mundo Real
+    description:  Aprenda desde o zero como construir websites modernos e robustos com Nuxt. Ou a melhorar o desempenho do seu website, qualidade do código, enquanto faz o melhor uso do framework.
     image: /img/video-courses/mastering_nuxt.png
     link: 'https://masteringnuxt.com/?utm_source=nuxt&utm_medium=link&utm_campaign=nsite'
-    tier: Premium
+    tier: Prémio
```

Comment:
@smarroufin Is the tier supposed to be translated?

Reply:
@ChristopheCVB what do you think about it, should it be translated?

Reply:
Yes, this is translatable, it's a label. All good!